src/site/blocks/promo.drupal.liquid

```diff
@@ -0,0 +1,39 @@
+{% comment %}
+"fieldPromo": {
+  "entity": {
+    "fieldImage": {
+      "entity": {
+        "image": {
+          "url": "http://vagovcms.lndo.site/sites/default/files/hub_promos/health-care.png",
+          "alt": null,
+          "title": null
+        }
+      }
+    },
+    "fieldPromoLink": {
+      "entity": {
+        "fieldLink": {
+          "uri": "https://www.telehealth.va.gov/",
+          "title": "VA Telehealth Services",
+          "options": null,
+          "url": {
+            "path": "https://www.telehealth.va.gov/"
+          }
+        },
+        "fieldLinkSummary": "Find out how we use telehealth technologies to provide specialty care and health monitoring to Veterans at their local VA clinic or in their own home."
+      }
+    }
+  }
+{% endcomment %}
+{% assign image = entity.fieldImage.entity.image %}
+<div class="merger-r-rail hub-promo" id="promo">
+    <img src="{{ image.url }}" alt="{{ image.alt }}" title="{{ image.title }}" width="{{ image.width }}" height="{{ image.height }}">
+    <section class="hub-promo-text">
```

Comment:
It looks like in the old template not all promo blocks have headings like this. Is this a change in how that content is modeled?